### PR TITLE
feat(Manager): Enhancements: Addressed Scroll Bar Issue and Introduced Environment Deletion Feature

### PR DESCRIPTION
**Pull request by nikhitamore**
## Explain the feature/fix

1. Adjustment of the page height to eliminate the redundant scroll bar triggered by the addition of the release note banner at the top of the Splaship Manager interface.
2. Implementation of functionality enabling users to delete ephemeral environments directly from the Splaship Manager, enhancing user control and management capabilities.

## Screenshot(s)

<details>
<summary>View Screenshots</summary>

for Ephemeral
image
image

and scroll bar

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/manager/src/services/persistent/queries.tsx [hunk 1]
```diff
@@ -81,3 +81,22 @@ export const useAddEnv = (propertyIdentifier?: string) => {
     }
   });
 };
+
+// POST OPERATIONS
+const deleteEphemeralEnv = async (dto: string): Promise<TDeleteEnvDTO> => {
```
Comment (soumyadip007) [inline packages/manager/src/services/persistent/queries.tsx:86]: Hi @nikhitamore , please change the `dto` to `envName` in future

File: packages/manager/src/views/WebPropertyDetailPage/components/Ephemeral/Ephemeral.tsx [hunk 2]
```diff
@@ -71,185 +78,224 @@ export const Ephemeral = ({ isSuccess, ephemeralEnvs }: Props): JSX.Element => {
...
-                  </EmptyStateBody>
-                </EmptyState>
-              </Bullseye>
-            </Td>
+            <Th textCenter>Environment Name</Th>
+            <Th width={15}>SPA name(s)</Th>
+            <Th textCenter>Created At</Th>
+            <Th textCenter>Action ID</Th>
+            <Th textCenter>Time left</Th>
+            {/* <Th textCenter /> */}
```
Comment (soumyadip007) [inline packages/manager/src/views/WebPropertyDetailPage/components/Ephemeral/Ephemeral.tsx:106]: please remove this line

**Review by soumyadip007 — APPROVED** [review]
Please make the changes

File: packages/manager/src/views/WebPropertyDetailPage/components/Ephemeral/Ephemeral.tsx (hunk 2 above)
Comment (nikhitamore) [reply packages/manager/src/views/WebPropertyDetailPage/components/Ephemeral/Ephemeral.tsx]: done

File: packages/manager/src/services/persistent/queries.tsx (hunk 1 above)
Comment (nikhitamore) [reply packages/manager/src/services/persistent/queries.tsx]: done

**Pull request merged**